geoportal/geoportailv3_geoportal/static-ngeo/js/measure/MeasureController.js

```diff
@@ -104,14 +144,20 @@ const exports = function($scope, $q, $http, $compile, gettext,
       fill: new olStyleFill({
         color: 'rgba(255, 204, 51, 0.3)'
       })
+    }),
+    text: new olStyleText({
+      font: 'bold 12px Calibri,sans-serif',
+      fill: new olStyleFill({
+        color: 'rgb(0 0 0 / 100)'
```

Comment:
I don't know this notation, what does it do?

Reply:
Should have been `rgb(0 0 0 / 100%)` (or even w/o the `/ 100%`, the default), see https://mobile.twitter.com/mathias/status/1253242715304857601 for ref.